### ISSUE TITLE
Add list of commits by gsoc applicants

### ISSUE THREAD

**Issue by jayvdb** [bekicot/gsoc-prep-tasks#9]
Under each org, it would be useful to be able to see all commits relevant to the GSOC applicants, when an org has a public GSOC applicant list.

This could be achieved using a [visualisation of all PRs](https://github.com/search?l=&q=is%3Apr+author%3Ajayvdb%20author%3Abekicot&ref=advsearch&type=Issues&utf8=%E2%9C%93) from a set of applicants, or a list of applicant [commits](https://github.com/search?l=&o=desc&q=author:bekicot&ref=advsearch&s=committer-date&type=Commits&utf8=%E2%9C%93).

**Comment by bekicot** [comment]
So it is like add manually a list of users and append it to search terms?
Because i dont see a way to do it automatically.

**Comment by jayvdb** [comment]
Yes, each org could add their list of applicants. I'll work on this one.

**Comment by jayvdb** [comment]
OK, here is one example: http://tinyurl.com/coala-gsoc2017pr

It only includes PR activity, as issue activity tends to be distracting rather than helpful.
It could also include `is:merged` to match this issues original concept of being a list of commits only.
